src/doc.c

```diff
@@ -2325,6 +2327,40 @@ void highlightText(Scope *sc, Dsymbol *s, OutBuffer *buf, size_t offset)
                 }
                 break;
 
+            case '`':
+                if (inBacktick)
+                {   inBacktick = 0;
```

Comment:
aight. Ironically, I used that style because I figured if I didn't, someone would complain about my style not being consistent!